include/RevProc.h

```diff
@@ -120,41 +120,50 @@ class RevProc{
     uint64_t cyclesIdle_MemoryFetch;
   };
 
+  ///< RevProc: Returns this proc's stats
   RevProcStats GetStats() { Stats.memStats = mem->memStats; return Stats; }
 
```

Comment:
If `RevProcStats` is a large structure, it might be faster to use a return type of `const RevProcStats&` instead of `RevProcStats` to avoid (or at least delay) expensive copies.